2833. Furthest Point From Origin/solution.py

```diff
@@ -0,0 +1,12 @@
+class Solution:
+    def furthestDistanceFromOrigin(self, moves: str) -> int:
+        # iterate through commands
+            # use switch-case statement to check for each command
```

Comment:
Sorry, this is actually a match-case statement, it's switch-case in JS and Java.